cluster/crds/pkg.crossplane.io_providerrevisions.yaml

```diff
@@ -79,10 +80,6 @@ spec:
                 description: DesiredState of the PackageRevision. Can be either Active
                   or Inactive.
                 type: string
-              essTLSSecretName:
```

Comment:
Do you have any concrete scenarios on how this could fail? 

I am planning to do some upgrade/rollback tests, please let me know if there is a specific case that I should consider.